include/links.php

```diff
@@ -59,6 +59,11 @@ public function __construct( &$polylang ) {
 	 */
 	public function get_home_url( $language, $is_search = false ) {
 		$language = is_object( $language ) ? $language : $this->model->get_language( $language );
-		return $is_search ? $language->search_url : $language->get_home_url();
+
+		if ( empty( $language ) ) {
+			return '';
+		}
```

Comment:
Done in [3210362](https://github.com/polylang/polylang/pull/1207/commits/321036243bab0496d8334bf6ba2ae38a1b49fd57).

Reply:
My goal was to simplify and finally 321036243bab0496d8334bf6ba2ae38a1b49fd57 introduces more complexity. Would you mind to revert?

And I propose to rewrite this `PLL_Links` method like this:
```php
public function get_home_url( $language, $is_search = false ) {
		if ( ! $language instanceof PLL_Language ) { 
				$language = $this->model->get_language( $language );
		}

		if ( empty( $language ) ) {
			return home_url( '/' );
		}

		return $is_search ? $language->get_search_url() : $language->get_home_url();
```
as checking the instance is better than just checking the object.
and returning the home url looks more logical than return nothing in case of erroneous param.

Reply:
Done in [43d79da](https://github.com/polylang/polylang/pull/1207/commits/43d79dae829fef09e06a3cedfa8e433750eb0e4a) and [f8d3a33](https://github.com/polylang/polylang/pull/1207/commits/f8d3a33388def1d679705b52f72d11a7906a93f5).